client/src/modules/reports/generate/inventory_report/inventory_report.html

```diff
@@ -31,37 +31,12 @@ <h3 translate>REPORT.STOCK.TITLE</h3>
             on-change="ReportConfigCtrl.onSelectDate(date)">
           </bh-date-editor>
 
-          <!-- delay -->
-          <div class="form-group col-xs-12 col-sm-6"
-            ng-class="{ 'has-error' : ConfigForm.$submitted && ConfigForm.delay.$invalid }">
-
-            <label class="control-label" translate>FORM.LABELS.DELAY</label>
-            <div class="input-group">
-              <input class="form-control" type="number" bh-integer name="delay" ng-model="ReportConfigCtrl.delay">
-              <span class="input-group-addon" translate>FORM.LABELS.MONTH</span>
-            </div>
-
-            <div class="help-block" ng-messages="ConfigForm.delay.$error" ng-show="ConfigForm.$submitted">
-              <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
-            </div>
-          </div>
-
-          <!-- purchase interval -->
-          <div class="form-group col-xs-12 col-sm-6"
-            ng-class="{ 'has-error' : ConfigForm.$submitted && ConfigForm.purchase_interval.$invalid }">
-
-            <label class="control-label" translate>FORM.LABELS.PURCHASE_INTERVAL</label>
-            <div class="input-group">
-              <input class="form-control" type="number" bh-integer name="purchase_interval" ng-model="ReportConfigCtrl.purchaseInterval">
-              <span class="input-group-addon" translate>FORM.LABELS.MONTH</span>
-            </div>
-
-            <div class="help-block" ng-messages="ConfigForm.purchase_interval.$error" ng-show="ConfigForm.$submitted">
-              <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
-            </div>
-          </div>
-
-          <div class="clearfix"></div>
+          <bh-yes-no-radios
+            label="STOCK.INCLUDE_EMPTY_LOT"
+            value="ReportConfigCtrl.includeEmptyLot"
+            help-text="STOCK.INCLUDE_EMPTY_LOT_HELP"
```

Comment:
These translations are missing

Reply:
This is true - they should be added with https://github.com/IMA-WorldHealth/bhima/pull/4086